lib/container.js

```diff
@@ -121,6 +125,35 @@ class Container {
     container.support = newSupport || {};
     container.plugins = newPlugins || {};
     container.translation = loadTranslation();
+    container.workerObjects = {};
+  }
+
+  /**
+   * Add worker object
+   *
+   * @api
+   * @param {Worker} workerObj
+   */
+  static addWorker(workerObj) {
+    container.workerObjects[workerObj.threadId] = workerObj;
+    invokeWorkerListeners(workerObj);
+  }
+
+  /**
+   * Share data across worker threads
+   *
+   * @api
+   * @param {Object<string, *>} data
+   */
+  static share(data) {
```

Comment:
please add a second parameter `options`, if `options.local === true` we won't share data accross workers.